third_party/ijar/ijar.cc

```diff
@@ -368,9 +370,12 @@ static void OpenFilesAndProcessJar(const char *file_out, const char *file_in,
             strerror(errno));
     abort();
   }
-  u8 output_length =
-      in->CalculateOutputLength() +
```

Comment:
I wonder of `CalculateOutputLength` should be handling this instead of fixing up the size later?

Reply:
> Consider adding a test?

Done

> I also commented in the bug--it's not clear if this is something that happened in a real build, it should be rare to see completely empty jar files that don't even contain a manifest.

True that.

Reply:
`CalculateOutputLength` is part of `zip.cc` and has no knowledge that we're adding `dummy` file.